import-uk

```diff
@@ -6,9 +6,14 @@
 
 set -e
 
-source ./find-mapit-venv
-
-export MANAGE="/usr/local/bin/govuk_setenv mapit $MAPIT_VENV_DIR/bin/python /var/apps/mapit/manage.py"
+if [ -f /.dockerenv ]; then
+  # Docker environment
+   export MANAGE="/govuk/mapit/manage.py"
+else
+  # Non-Docker environment
+  source ./find-mapit-venv
+  export MANAGE="/usr/local/bin/govuk_setenv mapit $MAPIT_VENV_DIR/bin/python /var/apps/mapit/manage.py"
+fi
```

Comment:
I think we should put this code in a file like `find-mapit-managepy` and then in both of these files we can just do `source ./find-mapit-managepy` rather than needing to duplicate the code.

Reply:
Thanks @thomasleese , I have updated this in https://github.com/alphagov/mapit-scripts/pull/8/commits/12f36f39c1498d1dc9a6d9f508bc23dab704f62e